frontend/src/components/LoadedPoll.tsx

```diff
@@ -0,0 +1,41 @@
+import { Poll } from '../client';
+import { VoteClient } from '../hooks/useVoteClient';
+import { useState } from 'react';
+import { Typography } from '@mui/material';
+import QuestionVoter from './QuestionVoter';
+import QuestionResults from './QuestionResults';
+
+interface LoadedPollProps {
+  poll: Poll;
+  voteClient: VoteClient;
+}
+
+export default function LoadedPoll(props: LoadedPollProps) {
+  const [questionIndex] = useState(0);
```

Comment:
This means that we currently don't support more than one question per poll, if I understand it correctly?

Reply:
Thats the Idea for the MVP, but this can later be included